kernel/kernel-api/src/main/java/io/delta/kernel/TableManager.java

```diff
@@ -43,5 +46,17 @@ static SnapshotBuilder loadSnapshot(String path) {
     return new SnapshotBuilderImpl(path);
   }
 
-  // TODO: static CreateTableTransactionBuilder buildCreateTableTransaction(...)
+  /**
+   * Creates a {@link CreateTableTransactionBuilder} to build a create table transaction.
+   *
+   * @param path the file system path for the delta table being created
+   * @param engineInfo information about the engine that is making the updates.
+   * @param schema the schema for the delta table being created
+   * @return create table builder instance to build the transaction
+   * @since 3.4.0
+   */
+  static CreateTableTransactionBuilder buildCreateTableTransaction(
+      String path, String engineInfo, StructType schema) {
```

Comment:
nit: can this order be `path, schema, engineInfo` -- plus that is what aligns with the constructor below